File: FitnessCentar/FitnessCenatr.Model/OdgovoriNaKomentare.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FitnessCentar.Model
{
    public class OdgovoriNaKomentare
    {
        public int Id { get; set; }

        public int? KomentarId { get; set; }

        public int? TrenerId { get; set; }

        public string? Tekst { get; set; }

        public DateTime? DatumOdgovora { get; set; }

      

    }
}

File: FitnessCentar/UI/fitness_admin/lib/models/odgovoriNaKomentare.dart
import 'package:json_annotation/json_annotation.dart';

part 'odgovoriNaKomentare.g.dart';

@JsonSerializable()
class OdgovoriNaKomentare{
       int? id;
       int? komentarId;
       int? trenerId;
       String? tekst;
       DateTime? datumOdgovora;
    
  
         OdgovoriNaKomentare(this.id,this.komentarId,this.trenerId,this.tekst,this.datumOdgovora);

  factory OdgovoriNaKomentare.fromJson(Map<String, dynamic> json) =>
      _$OdgovoriNaKomentareFromJson(json);

  Map<String, dynamic> toJson() => _$OdgovoriNaKomentareToJson(this);
}

